### PR TITLE
Add CSS morning challenge link to README, add CSS resources

### PR DESCRIPTION
**Pull request by njsfield**
Related #24

### CODE REVIEW

File: coursebook/week-1/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@
 - 17:30 - 18:00 — [Project introduction](.project.md)
 
 ## Day 3
-- 10.00 - 11.00 — Morning Challenge - Build Grid System with BEM
+- 10.00 - 11.00 — [Morning Challenge - CSS Gallery Challenge](https://github.com/njsfield/css-gallery-challenge)
```
Comment (nelsonic) [inline coursebook/week-1/README.md]: 😍

**Comment by jsms90** [comment]
No BEM resources?

**Comment by njsfield** [comment]
Originally I didn't clock to put any BEM resources because it's a research topic. But yeah a reference should be held, I've added a resource

**Comment by jsms90** [comment]
@njsfield Yeah, fair point. I'm not sure whether that's too much help for their research topics or not :thinking: But I seem to remember that people spent a lot of week 1's research time trying to figure out merge conflicts for the first time :laughing: So maybe extra help in week 1 isn't so bad?

Besides, so long as the resources don't offer too much hand-holding, maybe it's ok to give them useful links that would help with the research topics? After all, research day is more about making something that will be useful for the project now, rather than just producing text....but I'm still not sure. What do you reckon?

**Comment by njsfield** [comment]
Agreed! The extra help is good, especially in the first week. Without the resource, their wouldn't be too much challenge in finding this BEM resource as it's the top result for most search engines... but it helps to iterate the importance of learning from the research rather than merely hunting it down

**Comment by jsms90** [comment]
@njsfield You know what, I think I was wrong to suggest that. Part of research day is about learning to find information as part of a [spike](http://www.scaledagileframework.com/spikes/). That's why I've followed Marko's lead and written [our research day file](https://github.com/foundersandcoders/master-reference/blob/a68aaad778a501a82f723154f1dbfeef6b7884dd/coursebook/week-1/research-afternoon.md) as 1. research 2. use your knowledge

Maybe we shouldn't give them a direct link to the BEM guide, any more than I would link them to a tutorial on how to write an accessible navbar?As you said, it's easy enough to find anyway, but we wouldn't want them to stop there with their research :thinking:

**Comment by njsfield** [comment]
Fair point, as long as resources that are discovered in research day should be stored by the cohort (like our notes repo)

**Comment by jsms90** [comment]
@njsfield I don't think the `notes` repo was very successful. It's probably unrealistic to expect people to bother branch, commit, push, pull requesting every time they wanna add a link in there.

I'm still working on [resource-sharing](https://github.com/foundersandcoders/resource-sharing) :wink:

For now, putting useful resources at the end of their readmes is probably enough?

File: coursebook/week-1/resources.md [hunk 2]
```diff
@@ -16,3 +11,12 @@
   + Intentionally hiding semantics with the `presentation` role"
 [BAD demo](https://www.w3.org/WAI/demos/bad/Overview.html)
 [Videos](http://www.uiaccess.com/accessucd/resources_videos.html#screen_readers) of people using assistive technologies & putting code into practice
+
+### CSS
+[The 'centering-a-div' trick](https://css-tricks.com/centering-percentage-widthheight-elements/)
```
Comment (bradreeder) [inline coursebook/week-1/resources.md]: Could you re-format this to be bullet-pointed? It's all appearing as one block for me.

Comment (njsfield) [inline coursebook/week-1/resources.md]: Cool cool, done

**Pull request merged**